Website/src/rendering/src/components/ShopCommon/Price.tsx

```diff
@@ -75,19 +39,14 @@ export const PriceReact = ({
 
   if (max) {
     return (
-      // TODO: Extract JSX logic into a const
-      <div className={`price ${altTheme && 'price-orange'} ${sizeL && 'price-large'}`}>
+      <div className={cssClass}>
         <span className="price-base">
-          ${formatCurrency(min)} - ${formatCurrency(max)}
+          {formatCurrency(min)} - {formatCurrency(max)}
```

Comment:
Maybe also extract this line into a const?

Reply:
I am not sure it's necessary since it effectively extracts the logic into a function.

Reply:
Hmm not sure what you mean exactly, I just suggested extracting it in order to be consistent with not having any logic in JSX return statements

Reply:
I mean that there is no logic in the JSX, there is only a function call which holds the logic. And I think that's ok. I also confirmed with @jflheureux